NMS-15221: Added installer option to reset the password (#6034)
OpenNMS/opennms: docs/modules/deployment/pages/core/admin-password-reset.adoc

[[reset-admin-password]]
= Reset the Web UI admin password

This section explains how to set the Web UI admin password using the {page-component-title} installation script.

> After a {page-component-title} installation, the Web UI admin user's password defaults to `admin`.

Reviewer: ```suggestion
After you install {page-component-title}, the web UI admin user's password is set to `admin` by default.
```

> This password can be altered after the initial login to the {page-component-title} Web UI.

Reviewer: Slight change to come in line with our log in/sign in style. :)

```suggestion
You can change this password after you sign in to the web UI for the first time.
```

In the case of a forgotten password or deployment scenarios that require to set this password on installation you can use the {page-component-title} installation script to reset this password.

You need to add the option ` -R` (or `--reset-webui-password`) and providing the password to set as an argument.

> .Skip upgrade tasks and set the Web UI admin password to "foobar"
> [source, console]
> ----
> sudo /usr/share/opennms/bin/install -S -R foobar
> ----

Reviewer: ```suggestion
.Skip upgrade tasks and set web UI admin password to "foobar"
[source, shell]
sudo /usr/share/opennms/bin/install -S -R foobar
```
